pkg/pillar/cmd/verifier/verifier.go

```diff
@@ -118,7 +114,6 @@ func Run(ps *pubsub.PubSub) {
 		log.Fatal(err)
 	}
 	ctx.pubAppImgStatus = pubAppImgStatus
-	pubAppImgStatus.ClearRestarted()
```

Comment:
What ```ClearRestarted``` function do?

Reply:
Nothing any more. It used to do something when pubsub was just files (clear the restarted file) which was useful when testing was done by running individual agents in the build env on the laptop.
But on a device we always start without restarted, and we call SignalRestarted and get an HandleRestarted call in the subscriber.

So ClearRestarted is dead. (And so is other tidbits like the VerifyFilename in types/*.go)